api/api.go

```diff
@@ -34,9 +34,8 @@ type API struct {
 	//lint:ignore U1000 unused
 	metricsagent *metrics.Agent
 	vocinfo      *vochaininfo.VochainInfo
-	censusMap    sync.Map
-
-	db db.Database
+	censusdb     *censusdb.CensusDB
+	db           db.Database // used for internal db operations
```

Comment:
This database is currently used by the wallet handler. But ideally the wallet handler would have its own db so this "general" database can be removed from the API package.